types/react-dom/v15/index.d.ts

```diff
@@ -15,7 +15,7 @@ import {
     DOMAttributes, DOMElement
 } from 'react';
 
-export function findDOMNode<E extends Element>(instance: ReactInstance): E;
+export function findDOMNode<E extends Element>(instance: ReactInstance | null | undefined): E;
```

Comment:
This is one strange overload. It's one of the things the README tells you not to do (it's just a hidden `as` cast)... but it's already there 😔